tests/pytests/test_mdis_drivers.py

```diff
@@ -1,125 +1,473 @@
 import pytest
 import pvl
+import os
+import subprocess
+import numpy as np
+import spiceypy as spice
 
 import ale
-from ale.drivers import messenger_drivers
 from ale.base import data_naif
-from ale.base import label_pds3
-from ale.base import label_isis
 
-from unittest.mock import PropertyMock, patch
-
-# 'Mock' the spice module where it is imported
-from conftest import SimpleSpice, get_mockkernels
-
-simplespice = SimpleSpice()
-
-data_naif.spice = simplespice
-messenger_drivers.spice = simplespice
-label_pds3.spice = simplespice
+# We have to re-assign spiceypy because other tests overwrite it
+ale.base.data_naif.spice = spice
 
 from ale.drivers.messenger_drivers import MessengerMdisPds3NaifSpiceDriver
-from ale.drivers.messenger_drivers import MessengerMdisIsisLabelNaifSpiceDriver
 
-MessengerMdisPds3NaifSpiceDriver.metakernel = get_mockkernels
-MessengerMdisIsisLabelNaifSpiceDriver.metakernel = get_mockkernels
+@pytest.fixture(scope="module", autouse=True)
+def convert_kernels():
+    ale_root = os.path.split(ale.__file__)[0]
+    data_root = os.path.join(ale_root, '../tests/pytests/data/EN1072174528M')
+    kernels = [
+        'mdisAddendum009.ti',
+        'msgr_art_cal_EN1072174528M.xc',
+        'msgr_mdis_v160.ti',
+        'msgr_v231.tf',
+        'pck00010_msgr_v23.tpc',
+        'messenger_2548.tsc',
+        'msgr_gm_EN1072174528M.xc',
+        'msgr_sc_EN1072174528M.xc',
+        'msgr_merc_EN1072174528M.xsp',
+        'msgr_merc_bc_EN1072174528M.xsp',
+        'msgr_sun_EN1072174528M.xsp',
+        'msgr_sc_EN1072174528M.xsp',
+        'naif0012.tls'
+    ]
+    kernels = [os.path.join(data_root, kern) for kern in kernels]
+    bin_kernels = []
+    final_kernels = []
+    for kernel in kernels:
+        split_kernel = os.path.splitext(kernel)
+        if split_kernel[1] == '.xc':
+            subprocess.call(['tobin', os.path.join(data_root, kernel)])
+            bin_kernel = split_kernel[0] + '.bc'
+            bin_kernels.append(bin_kernel)
+            final_kernels.append(bin_kernel)
+        elif split_kernel[1] == '.xsp':
+            subprocess.call(['tobin', os.path.join(data_root, kernel)])
+            bin_kernel = split_kernel[0] + '.bsp'
+            bin_kernels.append(bin_kernel)
+            final_kernels.append(bin_kernel)
+        else:
+            final_kernels.append(kernel)
+    spice.furnsh(final_kernels)
+    yield
+    spice.unload(final_kernels)
```

Comment:
can unload error? I'm concerned about not cleaning up properly when this is done